src/main/java/io/github/moulberry/notenoughupdates/util/SBInfo.java

```diff
@@ -431,15 +431,15 @@ public void tick() {
 		}
 	}
 
-	public static JsonObject getCurrentMayor() throws ExecutionException, InterruptedException {
-		CompletableFuture<JsonObject> hypixelApiAsync = NotEnoughUpdates.INSTANCE.manager.hypixelApi.getHypixelApiAsync(
+	public void getCurrentMayor() throws ExecutionException, InterruptedException {
```

Comment:
Do we still throw those exceptions? Also i feel like we could give this function a better name now that it doesnt return anything anymore